config/sync/field.field.node.stanford_page.su_page_banner.yml

```diff
@@ -13,7 +13,7 @@ field_name: su_page_banner
 entity_type: node
 bundle: stanford_page
 label: 'Top Banner'
-description: "The top banner displays directly below the navigation and on interior pages, above the page title."
+description: 'The top banner displays directly below the navigation and on interior pages, above the page title.'
```

Comment:
I wonder why the quote type change. :man_shrugging:

Reply:
Somewhere a quark is bouncing around and influencing this world so that changes like this annoy developers for the quark's amusement.

Reply:
![noid](https://media1.tenor.com/images/97eec5182b0f94d210bab4cce79558ee/tenor.gif?itemid=15155572)